src/multi_processing/creator.py

```diff
@@ -0,0 +1,133 @@
+import time
+from multi_processing import pool_tasks
+
+
+class Creator:
+    """ A class to coordinate the creation of domain objects as per jobs
```

Comment:
I don't understand what you mean by "as per jobs observed within a multiprocessed queue"